File: src/main/java/com/core/base/corebase/repository/AccountRepository.java
package com.core.base.corebase.repository;

import com.core.base.corebase.domain.user.Account;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface AccountRepository extends JpaRepository<Account, UUID> {
    boolean existsByUid(UUID uuid);
}

File: src/main/kotlin/com/core/base/corebase/repository/AccountRepository.kt
package com.core.base.corebase.repository

import com.core.base.corebase.domain.user.Account
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface AccountRepository : JpaRepository<Account, UUID> {
    fun existsByUid(uuid: UUID): Boolean
}